### ISSUE TITLE
[web] Fonts not downloading in parallel with canvaskit.

### ISSUE THREAD

**Issue by ditman** [flutter/flutter#116223]
> @alanwutang11 just landed a commit in the Engine that will parallelize the download of canvaskit.js and the fonts that are required to start the app...

hello, i test latest version of flutter engine but it did not take effect.

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.3.8, on macOS 11.6 20G165 darwin-x64, locale zh-Hans-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.1)
[✓] Connected device (3 available)
[✓] HTTP Host Availability
```

<img width="1640" alt="image" src="https://user-images.githubusercontent.com/6874049/201003450-3cce4fb1-c2c7-40cf-a49d-2b3cc0b3429f.png">

----

> @zjrwdl hey! try switching to the master channel using "flutter channel master" and see if the new change shows up?

<img width="484" alt="image" src="https://user-images.githubusercontent.com/6874049/201248149-c3e6e530-e4a7-42a5-aa34-0db053e94f9d.png">

<img width="577" alt="image" src="https://user-images.githubusercontent.com/6874049/201248035-edcea4f9-f062-46cd-8b94-2eaa1e9c19f8.png">

<img width="1572" alt="image" src="https://user-images.githubusercontent.com/6874049/201248015-46702796-2beb-42be-96ec-8be4d94e2904.png">

<img width="1003" alt="image" src="https://user-images.githubusercontent.com/6874049/201248273-c190aff0-dff9-46b0-98dd-0ebda8eabc2f.png">

_Originally posted by @zjrwdl in https://github.com/flutter/flutter/issues/76009#issuecomment-1309771650_

**Comment by ditman** [comment]
Hey @zjrwdl, I've created a new issue for this problem. Could you please produce a minimal reproduction example so we can look at this issue? In this case, your `pubspec.yaml` would be interesting, and how you're using Fonts within the app.

/cc @alanwutang11

**Comment by IdoBloch4** [comment]
Here you can see that my fonts are being downloaded only after all `firebase-` libraries and `main.dart.js`, and also after `canvaskit`:
<img width="922" alt="image" src="https://user-images.githubusercontent.com/42539470/209736097-203e9574-45d7-4c57-97cb-171c4394841b.png">

Here is my `pubspec.yaml` (with small censored names): [pubspec.txt](https://github.com/flutter/flutter/files/10311152/pubspec.txt)

and here is my `flutter doctor` result:  [flutter doctor.txt](https://github.com/flutter/flutter/files/10311159/flutter.doctor.txt)

Thanks 🙏

**Comment by alanwutang11** [comment]
> Here you can see that my fonts are being downloaded only after all `firebase-` libraries and `main.dart.js`, and also after `canvaskit`: <img alt="image" width="922" src="https://user-images.githubusercontent.com/42539470/209736097-203e9574-45d7-4c57-97cb-171c4394841b.png">
> 
> Here is my `pubspec.yaml` (with small censored names): [pubspec.txt](https://github.com/flutter/flutter/files/10311152/pubspec.txt)
> 
> and here is my `flutter doctor` result: [flutter doctor.txt](https://github.com/flutter/flutter/files/10311159/flutter.doctor.txt)
> 
> Thanks 🙏

Hi @IdoBloch4, from your screenshot, it seems that the behavior from the [PR](https://github.com/flutter/engine/pull/36813) to download fonts concurrently with WASM is correct. Fonts should be downloaded in parallel with canvaskit.wasm, which is happening according to the waterfall graph.

**Comment by yjbanov** [comment]
Sounds like this is working as intended. We can probably improve it further but let's file a new issue for that.
